app/react/Metadata/components/MetadataForm.js

```diff
@@ -26,6 +26,7 @@ const selectTemplateOptions = createSelector(
   templates => templates.map(tmpl => ({ label: tmpl.get('name'), value: tmpl.get('_id') }))
 );
 
+// eslint-disable-next-line import/exports-last
```

Comment:
Please do not add eslint disable directives.

Reply:
Removed